staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,3 +109,24 @@ func (a *AdmissionOptions) Validate() []error {
 	errs := []error{}
 	return errs
 }
+
+// getEnabledPluginNames makes use of RecommendedPluginOrder and DefaultOffPlugins fields
+// to prepare a list of plugin names that are enabled.
+func (a *AdmissionOptions) getEnabledPluginNames() []string {
```

Comment:
nit: no get prefix in Go

Reply:
it's just a methodName, not a getter/setter of a field/property
https://golang.org/doc/effective_go.html#Getters

Reply:
still. The go style drop the prefix.

Reply:
this is the first time I came across this, it this idiomatic Go ?

Reply:
The reference I found is this: https://golang.org/doc/effective_go.html#Getters. But it's about getters only. If it is not a trivial getter, but does some computation, it's not clear from that link. Though it feels strange to make a difference between getters and other non-mutating methods. After all it's a hidden implementation detail what the actual state is of an object. Why not store `enabledPluginNames` in your case. Then it suddenly is a getter and the rule applies.